src/input/type_output.cc

```diff
@@ -3,6 +3,8 @@
  */
 
 #include "input/type_output.hh"
+#include "input/type_repository.hh"
+#include "rev_num.h"

```

Comment:
This include is forbidden for every  source else then main.cc. 
All information have to be passed through appropriate method of OutputJSON.